boot/bootutil/src/bootutil_priv.h

```diff
@@ -179,7 +187,13 @@ _Static_assert(BOOT_IMAGE_NUMBER > 0, "Invalid value for BOOT_IMAGE_NUMBER");
 #endif
 
 /** Maximum number of image sectors supported by the bootloader. */
```

Comment:
looks like this comment has peeled off what it describes

Reply:
Ah, this was already wrong before this PR, I will fix it.

Reply:
Should be fixed now!